src/command.directive.ts

```diff
@@ -84,7 +85,23 @@ export class CommandDirective implements OnInit, OnDestroy {
 
 	@Input(`${SELECTOR}Params`) commandParams: unknown | unknown[];
 
+	@Input("disabled")
+	set disabledInput(value: boolean) {
+		this.disabled = value;
+	}
+
+	@HostBinding("attr.disabled")
+	get disabled(): boolean | undefined { return this._disabled; }
+	set disabled(value: boolean | undefined) {
+		const disabled = value ? value : undefined;
+		if (value === this._disabled) {
+			return;
+		}
```

Comment:
```suggestion
		if (disabled === this._disabled) {
			return;
		}
```
not sure if intended tho false won't match

Reply:
about to push another, 1s (haven't set upstream to my fork yet 😆 )